db/data/20220112012521_add_default_facility_size.rb

```diff
@@ -0,0 +1,9 @@
+class AddDefaultFacilitySize < ActiveRecord::Migration[5.2]
+  def up
+    Facility.where(facility_size: nil).update_all(facility_size: :community)
```

Comment:
Don't think we can do this, there are larger facilities that don't have sizes set as well. This will also run in all envs, so it'll be messy to do based on facility_type. Wdyt about just fixing this manually on prod envs via the app console?

Reply:
We talked about this and the console is the right answer. We'll fix that before we deploy this.

Reply:
Sizes have been set via the console, this should be safe to run now